spec/rubocop/cop/rails/enum_hash_spec.rb

```diff
@@ -10,7 +10,7 @@
       it 'registers an offense' do
         expect_offense(<<~RUBY)
           enum status: %i[active archived]
-          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Enum defined as an array found in `status` enum declaration. Use hash syntax instead.
+                       ^^^^^^^^^^^^^^^^^^^ Enum defined as an array found in `status` enum declaration. Use hash syntax instead.
```

Comment:
`enum status: ` is not a bad part, so I think the cop should add an offense only to the array.

Reply:
Nice improvement ❤️